src/pages/event/[eventId].tsx

```diff
@@ -0,0 +1,114 @@
+import Head from "next/head";
+import Image from "next/image";
+import { EventHeader } from "@src/components/Header";
+
+const EventPage = ({
+    eventId
+}: InferGetStaticPropsType<typeof getStaticProps>) => {
+    const { data: event } = api.event.byId.useQuery({ id: eventId });
+    const { data: club } = api.club.byId.useQuery({ id: event ? event.clubId : "" });
+
+    if (!event) {
+        return <div>
+            Event Not Found.
+        </div>
+    } else if (!club) {
+        return <div>
+            Club Does Not Exist.
+        </div>
+    }
+
+    return <>
+        <Head>
+            <title>{event.name} - Jupiter</title>
+            <meta name="description">{event.name} - Jupiter</meta>
+        </Head>
+        <main className="w-full md:pl-72">
+            <EventHeader />
+            <div className="mb-5 flex flex-col space-y-6 px-7">
+                <div className="w-full h-full rounded-xl bg-slate-50 shadow-lg p-10 relative">
+                    <Image
+                        src={club.image}
+                        width={100}
+                        height={100}
+                        alt="club image"
+                    />
+                    <hr className="border-0 bg-black h-[1px] my-3 mt-5" />
+                    <div className="w-full flex justify-between mb-3">
+                        <div>
+                            <a className="font-semibold text-4xl" href={`/directory/${club.id}`}>{club.name}</a>
+                            <p className="font-semibold text-2xl">{event.name}</p>
+                        </div>
+                        <p className="font-semibold text-2xl">{new Date(event.startTime).toString().substring(0, 24)}</p>
+                    </div>
+                    <p>
+                        {event.description}
+                    </p>
+                    <div className="float-right mt-24">
+                        <button className="py-1 px-3 mr-3 rounded-lg border-black border-[1px] bg-white">
+                            Contact Us
+                        </button>
+                        <button className="py-1 px-3 rounded-lg border-black border-[1px] bg-white">
+                            Register
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </>;
+};
+export default EventPage;
+
+import { api } from "@src/utils/api";
+import { db } from "@src/server/db";
+import { appRouter } from "@src/server/api/root";
+import { createInnerTRPCContext } from "@src/server/api/trpc";
+import { createServerSideHelpers } from "@trpc/react-query/server";
+import { type GetServerSidePropsContext, type GetStaticPaths, type InferGetStaticPropsType } from "next";
```

Comment:
@RubenOlano Do we want all the imports at the top of the page?

Reply:
It should be fine since all the imports there relate the static props stuff